neqo-common/src/lib.rs

```diff
@@ -33,6 +33,15 @@ macro_rules! matches {
 
 #[must_use]
 pub fn hex(buf: &[u8]) -> String {
+    let mut ret = String::with_capacity(10 + buf.len() * 3);
```

Comment:
I think you can use a smaller capacity here, maybe just buf.len() * 2.

Reply:
Thanks. Changed.